### PR TITLE
changes to work with new pubsub headers

### PR DESCRIPTION
**Pull request by AdeHarrison**
# How to test?
Check out this branch
[Test as part of](https://github.com/ONSdigital/census-rm-case-processor/pull/26)

# Links
[Trello](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[Additional columns for Case events table ** (8)](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by LukeLoze1 — APPROVED** [review]
lgtm

File: acceptance_tests/features/steps/receipt.py [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 from google.api_core.exceptions import GoogleAPIError
 from google.cloud import pubsub_v1
 
 from acceptance_tests.utilities.date_utilities import convert_datetime_to_str
-from acceptance_tests.utilities.rabbit_context import RabbitContext
 from acceptance_tests.utilities.rabbit_helper import start_listening_to_rabbit_queue, store_all_msgs_in_context
 from config import Config
 
 
+@when("the receipt msg for a created case is put on the GCP pubsub")
```
Comment (liamtoozer) [inline acceptance_tests/features/steps/receipt.py:15]: Really picky, but to save introducing a new `@when` step here, could we instead just reuse the existing step description?

**Pull request merged**